library/Frontend/src/Juvix/Frontend/Sexp.hs

```diff
@@ -185,6 +187,33 @@ transGuardBody _alsee trans (Types.Body b) = trans b
 transGuardBody False trans (Types.Guard c) = transCond trans c
 transGuardBody True transs (Types.Guard c) = transCondMultiple transs c
 
+--------------------------------------------------------------------------------
+-- Effect Expansion
+--------------------------------------------------------------------------------
+
+transHand :: Types.Handler -> Sexp.T
+transHand (Types.Hand name ops ret) =
+  Sexp.list
+    [ Sexp.atom ":defhandler",
+      Sexp.atom (NameSymbol.fromSymbol name),
+      Sexp.listStar [Sexp.atom ":ops", Sexp.list (fmap transOperation ops)],
```

Comment:
What does `(:ops ops)` give us rather than doing it with just the second argument of defhandler being a list directly?

Reply:
`(:ops ops)` is more clearly structured than an anonymous list, also it's easier to query directly.